### ISSUE TITLE
v5 After restrart pinned tabs only load on some panels not others

### ISSUE THREAD

**Issue by AlexCzar** [mbnuqw/sidebery#1265]
## Steps to reproduce
- OS: macOS
- Hardware: M1 MacBook Pro 14"
- Open Firefox in [new profile](https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles)
- Install Sidebery
- have two panels
- pin tabs on both panels
- restart firefox

## Expected behavior
All pinned tabs load

## Actual behavior
Pinned tabs on one panel load, while on the other they remain unloaded. This issue appeared after update to v5

## Debug info

<details>
<summary><b>Addon data</b></summary>
     
```json
{
  "addonVersion": "5.0.0",
  "firefoxVersion": "117.0.1",
  "settings": {
    "nativeScrollbars": false,
    "nativeScrollbarsThin": true,
    "nativeScrollbarsLeft": false,
    "selWinScreenshots": false,
    "updateSidebarTitle": true,
    "markWindow": false,
    "markWindowPreface": "[Sidebery] ",
    "ctxMenuNative": false,
    "ctxMenuRenderInact": true,
    "ctxMenuRenderIcons": true,
    "ctxMenuIgnoreContainers": "",
    "navBarLayout": "hidden",
    "navBarInline": true,
    "navBarSide": "left",
    "hideAddBtn": false,
    "hideSettingsBtn": false,
    "navBtnCount": true,
    "hideEmptyPanels": false,
    "hideDiscardedTabPanels": false,
    "navActTabsPanelLeftClickAction": "none",
    "navActBookmarksPanelLeftClickAction": "none",
    "navTabsPanelMidClickAction": "discard",
    "navBookmarksPanelMidClickAction": "none",
    "navSwitchPanelsWheel": false,
    "subPanelRecentlyClosedBar": true,
    "subPanelBookmarks": true,
    "subPanelHistory": true,
    "groupLayout": "grid",
    "containersSortByName": false,
    "skipEmptyPanels": false,
    "dndTabAct": true,
    "dndTabActDelay": 750,
    "dndTabActMod": "none",
    "dndExp": "pointer",
    "dndExpDelay": 0,
    "dndExpMod": "none",
    "dndOutside": "win",
    "dndActTabFromLink": true,
    "dndActSearchTab": true,
    "dndMoveTabs": false,
    "dndMoveBookmarks": false,
    "searchBarMode": "dynamic",
    "searchPanelSwitch": "same_type",
    "searchBookmarksShortcut": "",
    "searchHistoryShortcut": "",
    "warnOnMultiTabClose": "collapsed",
    "activateLastTabOnPanelSwitching": true,
    "activateLastTabOnPanelSwitchingLoadedOnly": true,
    "switchPanelAfterSwitchingTab": "always",
    "tabRmBtn": "hover",
    "activateAfterClosing": "next",
    "activateAfterClosingStayInPanel": false,
    "activateAfterClosingGlobal": false,
    "activateAfterClosingNoFolded": true,
    "activateAfterClosingNoDiscarded": true,
    "askNewBookmarkPlace": false,
    "tabsRmUndoNote": true,
    "tabsUnreadMark": false,
    "tabsUpdateMark": "all",
    "tabsUpdateMarkFirst": true,
    "tabsReloadLimit": 5,
    "tabsReloadLimitNotif": true,
    "showNewTabBtns": true,
    "newTabBarPosition": "after_tabs",
    "tabsPanelSwitchActMove": false,
    "tabsPanelSwitchActMoveAuto": true,
    "tabsUrlInTooltip": "full",
    "newTabCtxReopen": false,
    "tabWarmupOnHover": true,
    "tabSwitchDelay": 0,
    "moveNewTabPin": "end",
    "moveNewTabParent": "last_child",
    "moveNewTabParentActPanel": false,
    "moveNewTab": "end",
    "moveNewTabActivePin": "start",
    "pinnedTabsPosition": "panel",
    "pinnedTabsList": false,
    "pinnedAutoGroup": false,
    "pinnedNoUnload": false,
    "tabsTree": true,
    "groupOnOpen": true,
    "tabsTreeLimit": 3,
    "autoFoldTabs": false,
    "autoFoldTabsExcept": "none",
    "autoExpandTabs": false,
    "autoExpandTabsOnNew": false,
    "rmChildTabs": "none",
    "tabsLvlDots": false,
    "discardFolded": false,
    "discardFoldedDelay": 0,
    "discardFoldedDelayUnit": "sec",
    "tabsTreeBookmarks": true,
    "treeRmOutdent": "branch",
    "ignoreFoldedParent": false,
    "colorizeTabs": false,
    "colorizeTabsSrc": "domain",
    "colorizeTabsBranches": false,
    "colorizeTabsBranchesSrc": "url",
    "inheritCustomColor": true,
    "hideInact": false,
    "hideFoldedTabs": false,
    "hideFoldedParent": "none",
    "nativeHighlight": false,
    "warnOnMultiBookmarkDelete": "collapsed",
    "autoCloseBookmarks": false,
    "autoRemoveOther": false,
    "highlightOpenBookmarks": false,
    "activateOpenBookmarkTab": false,
    "showBookmarkLen": false,
    "bookmarksRmUndoNote": true,
    "loadBookmarksOnDemand": true,
    "pinOpenedBookmarksFolder": true,
    "oldBookmarksAfterSave": "ask",
    "loadHistoryOnDemand": true,
    "fontSize": "m",
    "animations": true,
    "animationSpeed": "norm",
    "theme": "proton",
    "density": "default",
    "colorScheme": "ff",
    "sidebarCSS": false,
    "groupCSS": false,
    "snapNotify": true,
    "snapExcludePrivate": false,
    "snapInterval": 0,
    "snapIntervalUnit": "min",
    "snapLimit": 0,
    "snapLimitUnit": "snap",
    "snapAutoExport": false,
    "snapAutoExportType": "json",
    "snapAutoExportPath": "Sidebery/snapshot-%Y.%M.%D-%h.%m.%s",
    "snapMdFullTree": false,
    "hScrollAction": "switch_panels",
    "navSwitchPanelsDelay": 128,
    "scrollThroughTabs": "none",
    "scrollThroughVisibleTabs": false,
    "scrollThroughTabsSkipDiscarded": false,
    "scrollThroughTabsExceptOverflow": true,
    "scrollThroughTabsCyclic": false,
    "scrollThroughTabsScrollArea": 0,
    "autoMenuMultiSel": true,
    "multipleMiddleClose": false,
    "longClickDelay": 500,
    "wheelThreshold": false,
    "wheelThresholdX": 10,
    "wheelThresholdY": 60,
    "tabDoubleClick": "close",
    "tabsSecondClickActPrev": false,
    "tabsSecondClickActPrevPanelOnly": false,
    "shiftSelAct": true,
    "activateOnMouseUp": true,
    "tabLongLeftClick": "none",
    "tabLongRightClick": "none",
    "tabMiddleClick": "close",
    "tabMiddleClickCtrl": "discard",
    "tabMiddleClickShift": "duplicate",
    "tabCloseMiddleClick": "close",
    "tabsPanelLeftClickAction": "none",
    "tabsPanelDoubleClickAction": "tab",
    "tabsPanelRightClickAction": "menu",
    "tabsPanelMiddleClickAction": "tab",
    "newTabMiddleClickAction": "new_child",
    "bookmarksLeftClickAction": "open_in_act",
    "bookmarksLeftClickActivate": false,
    "bookmarksLeftClickPos": "default",
    "bookmarksMidClickAction": "open_in_new",
    "bookmarksMidClickActivate": false,
    "bookmarksMidClickRemove": false,
    "bookmarksMidClickPos": "default",
    "historyLeftClickAction": "open_in_act",
    "historyLeftClickActivate": false,
    "historyLeftClickPos": "default",
    "historyMidClickAction": "open_in_new",
    "historyMidClickActivate": false,
    "historyMidClickPos": "default",
    "syncName": "",
    "syncSaveSettings": false,
    "syncSaveCtxMenu": false,
    "syncSaveStyles": false,
    "syncSaveKeybindings": false,
    "selectActiveTabFirst": true
  },
  "permissions": {
    "allUrls": true,
    "webRequest": false,
    "webRequestBlocking": false,
    "proxy": false,
    "tabHide": false,
    "clipboardWrite": true,
    "history": false,
    "bookmarks": true,
    "downloads": false
  },
  "storage": {
    "size": "299 kb",
    "props": {
      "containers": "1.88 kb",
      "expandedBookmarkFolders": "55 b",
      "favDomains": "5.64 kb",
      "favHashes": "1.83 kb",
      "favicons_01": "261 kb",
      "profileID": "14 b",
      "settings": "4.81 kb",
      "sidebar": "1.85 kb",
      "tabsDataCache": "21.5 kb",
      "ver": "7 b"
    }
  },
  "sidebar": {
    "panels": {
      "bookmarks": {
        "type": 1,
        "id": "bookmarks",
        "name": "len: 9",
        "iconSVG": "icon_bookmarks",
        "iconIMGSrc": "",
        "iconIMG": "",
        "color": "toolbar",
        "lockedPanel": false,
        "tempMode": false,
        "skipOnSwitching": true,
        "rootId": "root________",
        "viewMode": "tree",
        "autoConvert": false,
        "srcPanelConfig": null
      },
      "firefox-default": {
        "type": 2,
        "id": "firefox-default",
        "name": "len: 7",
        "color": "toolbar",
        "iconSVG": "icon_tabs",
        "lockedPanel": false,
        "skipOnSwitching": false,
        "noEmpty": false,
        "newTabCtx": "none",
        "dropTabCtx": "none",
        "moveRules": [],
        "moveExcludedTo": -1,
        "bookmarksFolderId": -1,
        "newTabBtns": [],
        "srcPanelConfig": null
      },
      "G-R10EMgj5pm": {
        "type": 2,
        "id": "G-R10EMgj5pm",
        "name": "len: 4",
        "color": "red",
        "iconSVG": "dollar",
        "iconIMGSrc": "len: 39",
        "iconIMG": "len: 690",
        "lockedPanel": false,
        "skipOnSwitching": false,
        "noEmpty": false,
        "newTabCtx": "firefox-container-2",
        "dropTabCtx": "firefox-container-2",
        "moveRules": [
          {
            "id": "7eM0nv7maPWb",
            "active": true,
            "containerId": "firefox-container-2",
            "topLvlOnly": false
          },
          {
            "id": "VHgL-tAmaPWb",
            "active": true,
            "url": "purestorage.com"
          }
        ],
        "moveExcludedTo": -1,
        "bookmarksFolderId": -1,
        "newTabBtns": [],
        "srcPanelConfig": null
      }
    },
    "nav": [
      "bookmarks",
      "firefox-default",
      "G-R10EMgj5pm",
      "add_tp",
      "sp-0",
      "settings"
    ]
  },
  "containers": [
    {
      "id": "firefox-container-1",
      "cookieStoreId": "firefox-container-1",
      "name": "8",
      "icon": "...",
      "color": "blue",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "reopenRulesActive": false,
      "reopenRules": [],
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-2",
      "cookieStoreId": "firefox-container-2",
      "name": "4",
      "icon": "...",
      "color": "orange",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "reopenRulesActive": false,
      "reopenRules": [],
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-4",
      "cookieStoreId": "firefox-container-4",
      "name": "8",
      "icon": "...",
      "color": "pink",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "reopenRulesActive": false,
      "reopenRules": [],
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-6",
      "cookieStoreId": "firefox-container-6",
      "name": "8",
      "icon": "...",
      "color": "toolbar",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "reopenRulesActive": false,
      "reopenRules": [],
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-9",
      "cookieStoreId": "firefox-container-9",
      "name": "7",
      "icon": "...",
      "color": "green",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "reopenRulesActive": false,
      "reopenRules": [],
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-10",
      "cookieStoreId": "firefox-container-10",
      "name": "11",
      "icon": "...",
      "color": "pink",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "reopenRulesActive": false,
      "reopenRules": [],
      "userAgentActive": false,
      "userAgent": ""
    },
    {
      "id": "firefox-container-11",
      "cookieStoreId": "firefox-container-11",
      "name": "5",
      "icon": "...",
      "color": "red",
      "colorCode": "#37adff",
      "proxified": false,
      "proxy": null,
      "reopenRulesActive": false,
      "reopenRules": [],
      "userAgentActive": false,
      "userAgent": ""
    }
  ],
  "windows": [
    {
      "state": "maximized",
      "incognito": false,
      "tabsCount": 106
    }
  ],
  "bookmarks": {
    "bookmarksCount": 714,
    "foldersCount": 68,
    "separatorsCount": 3,
    "maxDepth": 4
  }
}
```
</details>

<details>
<summary><b>Logs</b></summary>

```
<nothing>
```
</details>

**Comment by mbnuqw** [comment]
Can reproduce. Cause is the workaround for https://github.com/mbnuqw/sidebery/issues/380

**Comment by Fred-Vatin** [comment]
On v5, only two of my pinned tabs load at startup. Some settings are greyed out when pinned tab position is not `in panel - top` (where they are invisible by the way).

![image](https://github.com/mbnuqw/sidebery/assets/22337329/e1b33306-42b7-426d-a787-5ee17bcf68e3)

![image](https://github.com/mbnuqw/sidebery/assets/22337329/f0801626-4c38-4d7a-8033-373a08fcb176)

**Comment by piec** [comment]
> Can reproduce. Cause is the workaround for #380

I can confirm the git version fixes it for me (loaded with firefox-developer-edition)

**Comment by Sharparam** [comment]
> On v5, only two of my pinned tabs load at startup.

I've had this issue for a while as well. After I manually switch to each pinned tab to load them, they stay loaded (I have "Prevent pinned tabs from unloading" enabled).

**Comment by xfzv** [comment]
> On v5, only two of my pinned tabs load at startup.

Same here, even after updating to [v5.1.0](https://github.com/mbnuqw/sidebery/releases/tag/v5.1.0)

Edit: with `Prevent pinned tabs from unloading` enabled, of course.

---

Edit 2 : seems to be fixed with [v5.1.1](https://github.com/mbnuqw/sidebery/releases/tag/v5.1.1) \o/

> Empty release to re-upload incorrectly uploaded version on AMO.

**Comment by mbnuqw** [comment]
Closing as fixed
